charts/xapp-anr/values.yaml

```
@@ -0,0 +1,68 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: ""
+    port: 0
+
+image:
+  repository: accelleran/xapp-anr
+  pullPolicy: IfNotPresent
+  tag: "0.1.0"
```

If we leave this out with an empty string, then the app version is used as the default value.

```suggestion
  tag: ""
```

---

Yes OK